src/components/VerificationTracker/AddPayroll.js

```diff
@@ -12,6 +12,7 @@ const AddPayroll = ({ userEnterprise }) => {
     [userEnterprise]
   );
   const { user } = useContext(AuthContext);
+  const enterprise = userEnterprise && userEnterprise.name;
```

Comment:
@rellfy I think this should be `const enterprise = userEnterprise?.name;`

Reply:
Sounds like a good idea. It works in Node 14, and in the more recent releases of modern browsers. Details and compatibility: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Both work, but I think using `?.` is clearer. Just need to be careful when it's running on Node that it's > v14, but since this is transpiled via babel it should support it.